go/login/login_test.go

```diff
@@ -0,0 +1,181 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package login
+
+import (
+	"encoding/hex"
+	"fmt"
+	"strconv"
+	"testing"
+
+	"github.com/google/glome/go/glome"
+)
+
+const (
+	tvsCnt = 2
+)
+
+var serviceKeyIds = []uint8{1, 0}
+
+func handleError(e error, t *testing.T) {
+	if e != nil {
+		t.Fatalf("Unexpected Error: " + e.Error())
+	}
+}
```

Comment:
I'm not a fan of error handling functions. Generally error checking is inlined, also because it can include more context. E.g. you can write `t.Fatalf("PrivateKeyFromSlice failed unexpectedly: %v", err)` and whoever reads the test result knows which function has failed. Generally Go does not print a backtrace because these are not exceptions so it's harder to figure out where an error came from.

Reply:
Removed.